### PR TITLE
Classroom update

### PR DESCRIPTION
**Pull request by suryap649**
- Cleaned up CPS OOD Example
- Removed all the images and cleaned up the text
- Please let me know if the text is intuitive to follow 
- All suggestions are welcome!

### CODE REVIEW

**Review by visionjo — APPROVED** [review]
@suryap649 , looks great!

I made a few minor changes.

Few things:
- Each section, add sentence or two explaining the contents of that section.
- Consider headings that are more uniform (i tweaked a little bit)
- Be sure to add {seealso} to pages that might be relevant

When you are ready and happy, okay with me to merge into devel

**Comment by suryap649** [comment]
@visionjo , I was wondering if removing the images and just having the text is intuitive enough for the users to follow through.

File: docs/source/classroom/cps_ood.md [hunk 1]
```diff
@@ -2,120 +2,90 @@
...
 # CPS Class Instructions
 
-:::{caution}
-**Note - The following instructions will only work for CPS classes.**
+:::{important}
+The following instructions will only work for CPS classes.
 :::
 
-These instructions describe the process of opening a CPS JupyterLab environment on the Open OnDemand (OOD) Discovery web portal and accessing class work directories.
+These instructions describe the process of opening a CPS JupyterLab environment on the {term}`Open OnDemand (OOD)` on the cluster web portal and accessing class `/work` folders.
```
Comment (joseph-cooke) [inline docs/source/classroom/cps_ood.md:9]: Courses are no longer using /work but they are using /courses.

File: docs/source/classroom/cps_ood.md [hunk 2]
```diff
@@ -2,120 +2,90 @@
...
 
-![Alt text](../images/cps-ood-menu.png)
+1. Select the default options and click **Launch**. Wait until the session is successfully created and ready to be launched (turns green).
 
-Select the default options and click **Launch**. Wait until the session is successfully created and ready to be launched (turns green).
+1. For more control of the session, modify **Time** for the session time (in hours), **Memory** to get more memory in GB, and the **Working Directory** where JupyterLab will launch.
 
-![Alt text](../images/cps-ood-jupyterform.png)
+    :::{note}
+    If **Working Directory** is left blank, the session will launch in the main class folder (in this example `/work/cps/ALY3070`). Alternatively, start the session directly from your personal working directory by entering: `/work/cps/ALY3070/students/[username]`, where `[username]` is your username on Discovery. The instructions below assume the field is left blank.
```
Comment (joseph-cooke) [inline docs/source/classroom/cps_ood.md:30]: I would clean this up and remove the explicit directories or pull an example from the notion course plan on how the directories would look. Thank you!!

File: docs/source/classroom/cps_ood.md [hunk 3]
```diff
@@ -2,120 +2,90 @@
...
+## Access Class Directories
 
-Navigate to the **students** directory, where you will see another directory under your username:
+1. After you are connected to a CPS JupyterLab session on OOD, you can access any shared class directories and your private class directory.
 
-![Alt text](../images/cps-ood-jupyterlab-students-folder.png)
+1. You can navigate between the class folders using the left menu. The files in the shared class directory are read ony and the students do not have permissions to edit or remove the files. Your instructor may use the shared class directory to share files related to the classwork
 
-Enter your personal class directory (here, username `mariana.levi` is shown):
+1. Navigate to your personal class directory under: `/students/<your username>`. Now you can create and edit Jupyter notebook files here.
```
Comment (joseph-cooke) [inline docs/source/classroom/cps_ood.md:45]: Just update to match the course plan in Notion. Thank you!!

File: docs/source/classroom/cps_ood.md [hunk 4]
```diff
@@ -2,120 +2,90 @@
...
+1. Click on the **Terminal** option under **Other** to open a Linux terminal.
 
-Navigate to your personal directory by typing the following command (change the class name from `ALY3070` to your class name accordingly):
+1. Navigate to your personal directory by typing the following command (change the class name from `ALY3070` to your class name accordingly).
 
-:::
-cd /work/cps/ALY3070/students/$USER
-:::
+    :::
+    cd /work/cps/ALY3070/students/$USER
```
Comment (joseph-cooke) [inline docs/source/classroom/cps_ood.md:72]: Update to match the course plan in Notion. Thank you!!

File: docs/source/classroom/cps_ood.md [hunk 5]
```diff
@@ -2,120 +2,90 @@
...
+1. Where `$USER` is a saved shell variable for your username. You can optionally also replace it with your username.
 
-Check that your assignment file is visible in the command line by typing `ls`. Then, Copy the assignment file to the **assignments** directory with this command (replace **Assignment1.ipynb** with your file name):
+1. Check that your assignment file is visible in the command line by typing `ls`. Then, copy the assignment file to the **assignments** directory with this command (replace **Assignment1.ipynb** with your file name).
 
-:::
-cp Assignment1.ipynb ../../assignments
-:::
+    :::
+    cp Assignment1.ipynb ../../assignments
```
Comment (joseph-cooke) [inline docs/source/classroom/cps_ood.md:80]: Not certain of the file structure for submission of CPS assignments under the new file tree.

File: docs/source/classroom/cps_ood.md [hunk 6]
```diff
@@ -2,120 +2,90 @@
...
-To remove an existing assignment, type (replace **Assignment1.ipynb** with your file name):
-
-:::
-rm ../../assignments/Assignment1.ipynb
-:::
+1. To remove an existing assignment, type (replace **Assignment1.ipynb** with your file name).
 
-Close the Terminal tab when done.
+    :::
+    rm ../../assignments/Assignment1.ipynb
```
Comment (joseph-cooke) [inline docs/source/classroom/cps_ood.md:86]: Not certain of the file structure for submission of CPS assignments under the new file tree.

**Comment by joseph-cooke** [comment]
@suryap649 - This looks really good. I just left some comments on updating paths and things like that to make the new course plan file tree. Thank you!!

File: docs/source/classroom/cps_ood.md (hunk 5 above)
Comment (suryap649) [reply docs/source/classroom/cps_ood.md:91]: Me either. I have posted a message regarding this on the documentation channel, we can change it as per response.

**Comment by suryap649** [comment]
@visionjo , @joseph-cooke , I was wondering if the page still looks intuitive enough without the images

**Comment by visionjo** [comment]
@suryap649

**Pull request merged**